nvi-commons/src/main/java/no/sikt/nva/nvi/common/model/business/Candidate.java

```diff
@@ -118,7 +118,7 @@ public static final class Builder {
         private List<ApprovalStatus> approvalStatuses;
         private List<Note> notes;
 
-        private Builder() {
+        public Builder() {
```

Comment:
Use the public static `builder()` method in stead of making this public:
`Candidate.builder().build()
`instead of
`new Candidate.Builder().build()`